econ

```diff
@@ -354,14 +354,21 @@ function post_build() {
     log_info "cp -R ${er}/data/landice/config ${rundir}/decks/${runid}/"
     cp -R ${er}/data/landice/config ${rundir}/decks/${runid}/
   fi
+  
+  echo ${STARTDIR}/${runid}
```

Comment:
Please use `log_info` or `log_warn`, not `echo`

Reply:
ahah yes, forgot to remove that line